util/storage-samples/storage-rv-sample/src/main/java/org/fidoalliance/fdo/storage/To0AllowListDenyListDbStorage.java

```diff
@@ -40,6 +48,12 @@
   public To0AllowListDenyListDbStorage(CryptoService cryptoService,
                                        DataSource dataSource) {
     super(cryptoService, dataSource);
+
+    entityManagerFactory = new EntityManagerFactoryFactory("org.fidoalliance.fdo.storage")
+        .withEntity(RvRedirects.class)
```

Comment:
Can we move this to hibernate.cfg.xml file ? So that we can configure from the DB properties from an XML file.